charts/kubewarden-controller/templates/webhooks.yaml

```diff
@@ -13,6 +52,7 @@ webhooks:
   - v1
   - v1beta1
   clientConfig:
+    caBundle: {{ $caCert }}
```

Comment:
When the old rootCA is around, I think it would still be nice to ensure it is part of this `caBundle`.
I **think** we might live without doing that for the kubewarden-controller, but there could be some race conditions.

I propose to define a new variable `caBundle` that contains the current rootCA and, eventually, the old one.